vault/config/overlays/nerc-ocp-infra/config/nerc-ocp-beta-test.yaml

```diff
@@ -0,0 +1,26 @@
+auth:
+- type: kubernetes
+  path: kubernetes/hypershift1
+  config:
+    kubernetes_ca_cert: "${ file `certs/letsencrypt.crt` }"
+    kubernetes_host: https://api.ocp-beta-test.nerc.mghpcc.org:6443
+    token_reviewer_jwt: "${ file `creds/nerc-ocp-beta-test-token-reviewer` }"
+  roles:
+  - bound_service_account_names:
+    - vault-secret-reader
+    bound_service_account_namespaces:
```

Comment:
Are you using a ClusterSecretStore in the cluster or multiple namespaced SecretStores?

Reply:
Multiple namespaced SecretStores